Build failures on ftgl dependency

I'm currently unable to build visualization.projectm due to a build dependency failure.  I've just cloned master, which successfully builds the freetype dependency, but the subsequent attempt to build ftgl, which depends on freetype, fails, complaining that freetype and freetype-config are not available (they are).  This is happening when attempting to build against the Kodi master branch. Not sure if the issue is with Kodi's build system, or visualization.projectm specifically.  The following is the output of my latest build attempt:

`[ 12%] Built target glm
[ 24%] Built target freetype
[ 26%] Performing build step for 'ftgl'
[ 12%] Performing update step for 'ftgl'
Preparing the FTGL build system...please wait

Found GNU Autoconf version 2.69
Found GNU Automake version 1.16.1
Found GNU Libtool version 2.4.6

Automatically preparing build ... done

The FTGL build system is now prepared.  To build here, run:
  ./configure
  make
[ 25%] Performing configure step for 'ftgl'
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking whether make supports the include directive... yes (GNU style)
checking for g++... g++
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking how to print strings... printf
checking for gcc... gcc
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for ANSI C header files... (cached) yes
checking for stdlib.h... (cached) yes
checking for wcsdup... yes
checking for strndup... yes
checking for X... libraries , headers 
checking for freetype-config... no
checking for FreeType -- version >= 9.0.3... no
configure: error: FreeType2 is required to compile this library
make[5]: *** [CMakeFiles/ftgl.dir/build.make:105: ftgl-prefix/src/ftgl-stamp/ftgl-configure] Error 1
make[4]: *** [CMakeFiles/Makefile2:73: CMakeFiles/ftgl.dir/all] Error 2
make[3]: *** [Makefile:130: all] Error 2
make[2]: *** [CMakeFiles/ftgl.dir/build.make:114: build/ftgl/src/ftgl-stamp/ftgl-build] Error 2
make[1]: *** [CMakeFiles/Makefile2:279: CMakeFiles/ftgl.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
`

--- a/depends/linux/ftgl/CMakeLists.txt
+++ b/depends/linux/ftgl/CMakeLists.txt
@@ -8,7 +8,7 @@ include(ExternalProject)
 externalproject_add(ftgl
                     SOURCE_DIR ${CMAKE_SOURCE_DIR}
                     UPDATE_COMMAND <SOURCE_DIR>/autogen.sh
-                    CONFIGURE_COMMAND ./configure --prefix=${OUTPUT_DIR} --with-pic --disable-shared --enable-static
+                    CONFIGURE_COMMAND ./configure --prefix=${OUTPUT_DIR} --with-pic --disable-shared --enable-static --with-ft-prefix=${OUTPUT_DIR}
                     BUILD_COMMAND make -C src
                     INSTALL_COMMAND ""
                     BUILD_IN_SOURCE 1)

